src/core_plugins/kibana/index.js

```diff
@@ -53,9 +53,11 @@ module.exports = function (kibana) {
           const configuredUrl = server.config().get('tilemap.url');
           const isOverridden = typeof configuredUrl === 'string' && configuredUrl !== '';
           const tilemapConfig = serverConfig.get('tilemap');
+          const vectormapsConfig = (serverConfig.get('vectormap')) ? serverConfig.get('vectormap') : { layers: [] };
```

Comment:
I was just thinking that you could use the [Joi .default](https://github.com/hapijs/joi/blob/v10.2.2/API.md#anydefaultvalue-description) functionality of the server schema as opposed to doing the ternary-if to default the `vectormapsConfig`